tutorial/README.md

```diff
@@ -88,35 +86,28 @@ Therefore for larger images, please use `bioformats2raw + raw2ometiff`.
 > by `raw2ometiff` - `LZW`, `zlib`, and `Uncompressed` as well as `jpeg` compression for 8 bit data. Support
 > for JPEG-2000 for >8 bit data is planned. Please open an issue if you would like this more immediately.
 
+If your OME-TIFF image has many [TIFF IFDs](https://en.wikipedia.org/wiki/TIFF#Multiple_subfiles), which correspond to indvidual time-z-channel sub-images, please generate an `offsets.json` file as well for remote HTTP viewing.
+This file contains the byte offsets to each IFD and allows fast interaction with remote data:
+
+ ```bash
+ $ pip install generate-tiff-offsets
+ $ generate_tiff_offsets --input_file Luca-7color_Scan1.ome.tif 
+ ```
+
+For viewing in Avivator, this file should live adjacent to the OME-TIFF file in its folder and will be automatically recognized and used.
+For use with Viv's loaders/layers, you need to fetch the `offsets.json` and pass it in as an argument to the [loader](http://viv.gehlenborglab.org/#createometiffloader).
+Please see [this sample](http://viv.gehlenborglab.org/#getting-started) for help getting started with using the Viv API.
```

Comment:
This index can be reused by other Viv-based applications and even clients in other languages to improve remote OME-TIFF performance. If using Viv, you must fetch the `offsets.json` directly in your application code. See [our example] (http://viv.gehlenborglab.org/#getting-started) for help getting started.


> ⚠️ IMPORTANT ⚠️ 
Avivator requires the `offsets.json` file to be adjacent to the OME-TIFF on the server in order to leverage this feature. For example, if an index is generated for the dataset in this tutorial, the following directory structure is correct:

```
data
├── LuCa-7color_Scan1.offsets.json
└── LuCa-7color_Scan1.ome.tif
```